### PR TITLE
Issue 589 590 595 fix

### PR DESCRIPTION
**Pull request by fraincs**
Fix for issues #589 #590 #595

### CODE REVIEW

File: packages/TagPicker/src/tag-picker.scss
Comment (franckgaudin) [inline packages/TagPicker/src/tag-picker.scss:1]: A regression has been introduced in the rebrand. When you type "just..." for example, the word shifts to the left and disappears under the tag on the side.

tag-regression

Comment (fraincs) [reply packages/TagPicker/src/tag-picker.scss:1]: IIRC it has been for a while, I'll look into it.

Comment (franckgaudin) [reply packages/TagPicker/src/tag-picker.scss:1]: 🙏

**Comment by franckgaudin** [comment]
Concerning issue #589 the color pastie is not correctly aligned with the text. 

combobox-align

**Comment by fraincs** [comment]
> Concerning issue #589 the color pastie is not correctly aligned with the text.
> 
> combobox-align

Good catch, this might not be linked to this PR but I'll add it to the PR.

**Comment by fraincs** [comment]
> Concerning issue #589 the color pastie is not correctly aligned with the text.
> 
> combobox-align

@franckgaudin can you provide a link as I don't see this issue on my end: 

image

**Comment by vicky-comeau** [comment]
> > Concerning issue #589 the color pastie is not correctly aligned with the text.
> > combobox-align
> 
> @franckgaudin can you provide a link as I don't see this issue on my end:
> 
> image

It's when the combobox dropdown is open:
image

**Review by franckgaudin — APPROVED** [review]

**Pull request merged**